JumpscaleLibs/clients/explorer/reservations.py

```diff
@@ -36,9 +41,7 @@ def _next_action(self, next_action):
                 raise j.exceptions.Input("next_action should be of type int")
         return next_action
 
-    def iter(
-        self, customer_tid=None, next_action=None,
-    ):
+    def iter(self, customer_tid=None, next_action=None):
```

Comment:
wouldnt it be better to place the filter at the bottom of this method? Not iter function is not patched properly

Reply:
done at https://github.com/threefoldtech/jumpscaleX_libs/pull/212/commits/23f72622f48cedfd72260a6461041299e5c61d13